precompiles/gmp/Gmp.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity >=0.8.3;
 
 /// @dev The Gmp contract's address.
-address constant GMP_ADDRESS = 0x0000000000000000000000000000000000000815;
```

Comment:
Think we need to run `npm run pre-build-contracts` so any test using this contract is also able to use this change.

Reply:
In our tests we used a constant defined in `constants.ts` IIRC instead of this one, which is probably why we didn't catch it in the first place. Maybe we should fix that...

Reply:
This issue is included in a Jira, btw